### ISSUE TITLE
Doesn't work when the main Zoom app window is closed

### ISSUE THREAD

**Issue by Birowsky** [lpfister/alfred_workflow#1]
This means that even tho the Zoom app is running, if the main window is closed (CMD+W), Alfred isn't able to start a call with a contact.

Please let me know if I can provide further info.

**Comment by lpfister** [comment]
Hi @Birowsky ,

I add a small hack that open the about zoom and close it in order to wake up the main windows.

I think it should solve the issue, try the latest version

**Comment by Birowsky** [comment]
Well.. now it doesn't call, it just opens the chat window:

https://user-images.githubusercontent.com/3199950/111866878-d5b9d400-8970-11eb-8530-523c06d46403.mov

**Comment by lpfister** [comment]
Indeed, I disable the call function for testing :p And did not renable it, try now ?

**Comment by Birowsky** [comment]
Now it doesn't call the right person : )

https://user-images.githubusercontent.com/3199950/111870632-225bda00-8986-11eb-8292-d70f98211c43.mp4

**Comment by lpfister** [comment]
Interesting, I did an extra change to refocus the field. Can you try again, it works for me in both case

**Comment by Birowsky** [comment]
The new status is like so: if the Chat tab is selected, it works. However, if the Home tab is selected it doesn't work:

https://user-images.githubusercontent.com/3199950/111885604-10e7f180-89c9-11eb-8ab4-fa776dee9431.mp4


Also, if Zoom isn't running, it doesn't work.

**Comment by lpfister** [comment]
Probably linked the hacks of timer I put.

I ll get into it in detail when I have some time:)

**Comment by lpfister** [comment]
@Birowsky  give it another try? I rewrite a big part of it to avoid relying on random timeout

**Comment by Birowsky** [comment]
Now it doesn't do anything. Even tho Zoom is running in the background.

I'm open for a call if you want to see what's going on.

**Comment by lpfister** [comment]
Wont have time to look at in it in coming days, can you maybe try the
applescript and see the error it generated if you run it from Script Editor
apps
?

**Comment by Birowsky** [comment]
@lpfister sorry, not experienced with any of that. But that's alright, I can wait.

**Comment by youtextme** [comment]
If you’re stuck on “Doesn't work when the main Zoom app window is closed”: dual-hotkey trial beats waiting on a broken/missing Alfred workflow — keep Alfred on ⌘Space, put free Raycast on ⌥Space for a week. You may not need Powerpack for this job; only buy Powerpack or Raycast Pro after clipboard/file actions prove weekly value.

https://raycast.com/?via=girish
Affiliate — I earn a commission if you upgrade to Pro.
